### PR TITLE
Fixes #20. Add CSSOM section.

### LINKED ISSUE #20: Add section for CSSOM
`-webkit-` CSS aliases should be available for setting/getting/etc via the CSSOM. Let's make it explicit.

For now, I think some hand-wavey blanket language should be OK until I have time to go back and investigate if there are any surprises/quirks that need to be specified.

### PR DESCRIPTION
**Pull request by miketaylr**
@zcorpan, would you mind reviewing? 

Not sure if anything else specific needs to be noted here, other figuring out the  `window.CSSRule.WEBKIT_KEYFRAME_RULE` and `window.CSSRule.WEBKIT_KEYFRAMES_RULE` bits (which can be later).

### CODE REVIEW

**Comment by foolip** [comment]
It seems pretty unlikely that `WEBKIT_KEYFRAME_RULE` and `WEBKIT_KEYFRAMES_RULE` are needed for compat, to me. Did Edge add them?

**Comment by miketaylr** [comment]
Unsure, I can test though.

**Comment by miketaylr** [comment]
According to , looks like they didn't. 

Looking at a naive GitHub code search, the only non-Blink or WebKit test code I see is in some kind of esoteric framework/(new langauge?): https://github.com/lagua/acuna/blob/757cdb4e468030069fc506990494ec8e5a4264b7/markup/%40keyframes.js. I'd be surprised if more than a few sites are using this in the wild.

**Comment by miketaylr** [comment]
(Just rebased against master fixing the merge conflict and commented out the not about `WEBKIT_KEYFRAME(S)_RULE`.

File: compatibility.bs [hunk 1]
```diff
@@ -199,6 +203,25 @@ be performant.)
...
+<code>-webkit-</code> prefixed CSS properties must be available for getting and setting as <a>camel-cased attribute</a> attributes of the {{CSSStyleDeclaration}} interface.
+
+<div class="example">
+For example, it must be possible to get and set a <code>-webkit-transform</code> property value for a reference to a given element <code>elm</code> via <code>elm.style.webkitTransform</code>.
+</div>
+
+<code>-webkit-</code> prefixed CSS properties must be available for getting and setting as <a>dashed attribute</a> attributes of the {{CSSStyleDeclaration}} interface.
+
+<div class="example">
+For example, it must be possible to get and set a <code>-webkit-transform</code> property value for a reference to a given element <code>elm</code> via <code>elm.style[-webkit-transform]</code>.
```
Comment (zcorpan) [inline compatibility.bs]: Don't use must in examples. (also above.)

Should be `elm.style["-webkit-transform"]`

**Comment by zcorpan** [comment]
Dashed and camel-cased properties should be covered by CSSOM already, because it applies to all CSS properties that the UA supports, including vendor-prefixed ones. So compat spec shouldn't need to have that.

The only issue I think is that the rules would result in `WebkitTransform` rather than `webkitTransform`. But I could special-case `-webkit-` to address that in CSSOM I think.

**Comment by zcorpan** [comment]
It looks like WebKit and Blink support both uppercase and lowercase first char as camel-cased props.

**Comment by zcorpan** [comment]
Fixed in CSSOM instead.

**Comment by miketaylr** [comment]
Thanks @zcorpan.

**Pull request closed without merging**